extensions/amp-story/1.0/media-pool.js

```diff
@@ -807,10 +807,17 @@ export class MediaPool {
     return this.enqueueMediaElementTask_(poolMediaEl, new PauseTask()).then(
       () => {
         if (rewindToBeginning) {
-          this.enqueueMediaElementTask_(
-            /** @type {!PoolBoundElementDef} */ (poolMediaEl),
-            new SetCurrentTimeTask({currentTime: 0})
-          );
+          // We add a 10 second delay to rewinding as sometimes this causes
+          // interlacing/glitch/frame jump when a new video is starting to play.
+          // A 0 delay isn't enough as we need to push the "seeking" event
+          // to the next tick of the event loop.
```

Comment:
Can you specifically mention that it's a hack and in an ideal world we'd do it different, fixing navigations animation frames?

Reply:
done.